### PR TITLE
Update Footer

### PR DESCRIPTION
**Pull request by guntupallikoushik1999**
Latest Update of Footer
image
image
image
image

### CODE REVIEW

**Comment by guntupallikoushik1999** [comment]
image
image
image
image
# update footer-1

File: src/components/footer/Footer.vue [hunk 1]
```diff
@@ -49,7 +92,221 @@ London EC1V 2NX
...
+    margin-top: 10px;
+  }
+}
+@media (max-width: 850px) {
+  ._1THa96r {
+    max-width: 100%;
+    padding: 0 15px;
+    width: auto;
+  }
+  ._1l86est {
```
Comment (haigopi) [inline src/components/footer/Footer.vue:127]: Please get the right names.

File: src/components/footer/Footer.vue [hunk 2]
```diff
@@ -1,43 +1,86 @@
 <template>
   <v-container fluid ma-0 pa-0>
     <v-footer dark padless>
-      <v-card
-        flat
-        tile
-        class="grey-blue lighten-1 white--text text-center"
-        style="width:100%"
-      >
+      <div class="Unfoiw2 _2JY6S0m">
```
Comment (haigopi) [inline src/components/footer/Footer.vue:4]: We should not write like this. 
Can we use Vue Cards to achieve the same?  We can reduce lot of code and style code as well.

Comment (guntupallikoushik1999) [reply src/components/footer/Footer.vue:4]: Ok

**Comment by haigopi** [comment]
Closing as already completed.

**Pull request closed without merging**